tests/conftest.py

```diff
@@ -260,3 +261,142 @@ def tiny_config(tmp_path):
         yaml.dump(cfg, out_file)
 
     return cfg_file
+
+
+@pytest.fixture
+def mgf_small_unannotated(tmp_path):
```

Comment:
**question:** Why is this necessary? Can't you use the existing MGF and just ignore the sequence keys?

Reply:
The name perhaps isn't the best. Not only does it remove the annotation, but it also adds a `SCANS=` field that is not present in the other `mgf` constructor. I thought making a new fixture was easier than possibly breaking some other tests in weird ways by modifying the working fixture.

Reply:
We shouldn't require in Casanovo that MGF files contain a SCANS field, as this is non-standard. Hence, the tests should be able to work without this as well, and the existing MGF can be reused.

Reply:
Fixed, uses existing `.mgf`